src/applications/personalization/appointments/actions/index.js

```diff
@@ -1,98 +1,13 @@
-import { replace, uniq } from 'lodash';
 import recordEvent from 'platform/monitoring/record-event';
-
-import environment from '~/platform/utilities/environment';
 import { apiRequest } from '~/platform/utilities/api';
-
 import moment from '~/applications/personalization/dashboard/lib/moment-tz';
-import {
-  getCCTimeZone,
-  getVATimeZone,
-  getTimezoneBySystemId,
-} from '~/applications/personalization/dashboard/utils/date-formatting/timezone';
 import {
   FETCH_CONFIRMED_FUTURE_APPOINTMENTS,
   FETCH_CONFIRMED_FUTURE_APPOINTMENTS_FAILED,
   FETCH_CONFIRMED_FUTURE_APPOINTMENTS_SUCCEEDED,
-  FUTURE_APPOINTMENTS_HIDDEN_SET,
-  VIDEO_TYPES,
 } from '~/applications/personalization/dashboard/constants';
-import MOCK_FACILITIES from '~/applications/personalization/dashboard/utils/mocks/appointments/MOCK_FACILITIES.json';
-
-import { vaosV2Helpers, getStagingID } from './utils';
-
-const CANCELLED_APPOINTMENT_SET = new Set([
-  'CANCELLED BY CLINIC & AUTO RE-BOOK',
-  'CANCELLED BY CLINIC',
-  'CANCELLED BY PATIENT & AUTO-REBOOK',
-  'CANCELLED BY PATIENT',
-]);
-
-function isVideoVisit(appt) {
-  return !!appt.vvsAppointments?.length;
-}
-
-function isCanceled(appt) {
-  return CANCELLED_APPOINTMENT_SET.has(
-    appt.attributes?.vdsAppointments?.[0]?.currentStatus,
-  );
-}
-
-/**
- * Finds the datetime of the appointment depending on the appointment type
- * and returns it as a moment object
- *
- * @param {string} type 'cc' or 'va'
- * @param {Object} appt VAR appointment object
- * @returns {Object} Returns appointment datetime as moment object
- */
-function getLocalAppointmentDate(type, appt) {
-  if (type === 'cc') {
-    const zoneSplit = appt.timeZone.split(' ');
-    const offset = zoneSplit.length > 1 ? zoneSplit[0] : '+0:00';
-    return moment
-      .utc(appt.appointmentTime, 'MM/DD/YYYY HH:mm:ss')
-      .utcOffset(offset);
-  }
-
-  const timezone = getTimezoneBySystemId(appt.facilityId)?.timezone;
-  const date = isVideoVisit(appt)
-    ? appt.vvsAppointments[0].dateTime
-    : appt.startDate;
-  return timezone ? moment(date).tz(timezone) : moment(date);
-}
 
-function isAtlasLocation(appointment) {
-  return appointment.attributes?.vvsAppointments?.some(
-    element => element.tasInfo,
-  );
-}
-
-function getAdditionalInfo(appointment) {
-  const appointmentKind =
-    appointment.attributes?.vvsAppointments?.[0]?.appointmentKind;
-
-  if (isAtlasLocation(appointment)) {
-    return 'at an ATLAS location';
-  }
-
-  if (appointmentKind === VIDEO_TYPES.clinic) {
-    return 'at a VA location';
-  }
-
-  if (appointmentKind === VIDEO_TYPES.gfe) {
-    return 'using a VA device';
-  }
-
-  if (
-    appointmentKind === VIDEO_TYPES.adhoc ||
-    appointmentKind === VIDEO_TYPES.mobile
-  ) {
-    return 'at home';
-  }
-
-  return null;
-}
+import { vaosV2Helpers } from './utils';
 
 export function fetchConfirmedFutureAppointmentsV2() {
```

Comment:
Since we are removing the V1 version altogether would this make sense to just rename `fetchConfirmedFutureAppointments` for clarity?